TLM/TLM/Manager/Impl/LaneConnection/LaneConnectionSubManager.cs

```diff
@@ -28,7 +28,7 @@ public class LaneConnectionSubManager :
 #pragma warning restore RAS0002 // Readonly field for a non-readonly struct
 
 #if DEBUG
-        private bool verbose_ = DebugSwitch.LaneConnections.Get();
+        private bool verbose_ => DebugSwitch.LaneConnections.Get();
```

Comment:
Can debug switch change during runtime?

Reply:
yes. you can reload the config file.
Actually I don't load the game with debug switch for faster loading speed and then change it runtime.